### PR TITLE
Add testing for the python 3.9 SAC container

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: bci_tester/data.py [hunk 1]
```diff
@@ -355,7 +354,7 @@ def create_BCI(
         if not baseurl:
             raise ValueError("Missing CONTAINER_URL for TARGET manual")
     elif OS_VERSION == "tumbleweed":
-        if bci_type in (ImageType.APPLICATION, ImageType.SAC_APPLICATION):
+        if bci_type in (ImageType.APPLICATION, ImageType.SUSE_APPLICATION):
```
Comment (dcermak) [inline bci_tester/data.py:357]:
```suggestion
        if bci_type in (ImageType.APPLICATION, ImageType.SUSE_APPCOLLECTION):
```

**Review by dcermak — DISMISSED** [review]
You have to change the python 3.9 container to `ImageType.SUSE_APPCOLLECTION` in [`tests/test_metadata.py:166`](https://github.com/SUSE/BCI-tests/pull/675/files#diff-f49ce5cf2ce23f6922b7add17ca149482d190f1c2b5892c5d1d55a28497c5556R166), otherwise the reference & url tests fail

**Review by dcermak — COMMENTED** [review]
LGTM, but now we have a new problem: we have two different python 3.11 containers with the **same** marker. This means that QA can no longer test explicitly for one of the two python 3.11 containers, only for both of them.

**Comment by dirkmueller** [comment]
> LGTM, but now we have a new problem: we have two different python 3.11 containers with the **same** marker. This means that QA can no longer test explicitly for one of the two python 3.11 containers, only for both of them.

I don't think thats any problem whatsoever..

**Review by alexandrevicenzi — APPROVED** [review]

**Comment by grisu48** [comment]
> I don't think thats any problem whatsoever..

That makes test issue debugging and problem pinpointing more difficult.
If we do this, we will increase the human-based latency in the release process.

**Comment by dirkmueller** [comment]
> That makes test issue debugging and problem pinpointing more difficult. If we do this, we will increase the human-based latency in the release process.

@grisu48 the container are essentially the same. I don't expect one failing and the other one succeeding. which means there is no additional overhead in test debugging.

**Pull request merged**